acceptance_tests/features/environment.py

```diff
@@ -39,6 +41,29 @@ def before_scenario(_, scenario):
         return
 
 
+def prepare_collection_exercises():
+    # for us028
```

Comment:
If it's for us028 I wonder if it would be easier to follow if it's set up in the feature given step, what do you think?